### DISCUSSION TITLE
6.1 kernel on EKS 1.24

### DISCUSSION THREAD

**Discussion by dmatias-onfido** [bottlerocket-os/bottlerocket#3593] [Q&A]
Hello,

We are starting to experiment with Bottlerocket and one key aspect that I wanted to test are the NVIDIA drivers (525/535) and I saw that they were [packaged in the Kernel 6.1](https://github.com/bottlerocket-os/bottlerocket/pull/3203). However, we are still running on EKS 1.24 for the moment and this kernel seems to be only available with the 1.28 variant. 
I wanted to know how to approach this ? Should I build myself the AMI ? Is there any limitations of 1.24 that would make it impossible to work on the kernel 6.1 ? 
Thank you 🙏

**Comment by yeazelm** [comment]
Thanks for cutting this @dmatias-onfido! We released 1.16.1 yesterday which upgraded the NVIDIA drivers for `aws-k8s-1.24`, `aws-k8s-1.25`, `aws-k8s-1.26`, and `aws-k8s-1.27` to 535. Can you try this latest release and see if this resolves your issue?

**Reply by dmatias-onfido** [reply]
That's awesome ! Thank you so much :D
